py/runtime0.h

```diff
@@ -101,6 +101,23 @@ typedef enum {
     // Operations below this line don't appear in bytecode, they
     // just identify special methods.
 
+    // MP_BINARY_OP_REVERSE_* must follow immediately after MP_BINARY_OP_*
+#if MICROPY_PY_REVERSE_SPECIAL_METHODS
+    MP_BINARY_OP_REVERSE_OR,
+    MP_BINARY_OP_REVERSE_XOR,
+    MP_BINARY_OP_REVERSE_AND,
+    MP_BINARY_OP_REVERSE_LSHIFT,
+    MP_BINARY_OP_REVERSE_RSHIFT,
+    MP_BINARY_OP_REVERSE_ADD,
+
```

Comment:
As said before, no need to separate these.

Reply:
Let's fix it consistently in a separate patch.